code/game/turfs/turf.dm

```diff
@@ -108,7 +108,7 @@
 //		O.update_gravity(O.mob_has_gravity(src))
 
 	var/loopsanity = 100
-	for(var/atom/A in range(1))
+	for(var/atom/movable/A in range(1))
```

Comment:
This was an emergency fix for server-crashing runtime spam so I didn't integrate it as thoroughly as I would have liked. Tiger has a point about changing it to atom/movable, it does fix the problem but it also means that it won't be possible for turfs to use HasProximity in the future. Ultimately it's not particularly relevant as no turfs currently use it as far as I know, but the point stands.